test/fixtures/compound20.css

```diff
@@ -0,0 +1,16 @@
+
+:root {
+	--color: #000000;
+}
+
+body.bar * {
+	--color: #ffffff;
+}
+
+body.bar *:hover {
+	--color: #111111;
+}
```

Comment:
This looks like a known-limitation and should be in the output. Let's add a comment